plugins/guests/darwin/cap/mount_vmware_shared_folder.rb

```diff
@@ -81,9 +81,15 @@ def self.mount_vmware_shared_folder(machine, name, guestpath, options)
                   content = @apply_firmlinks[machine.id][:content].join("\n")
                   # Write out the synthetic file
                   comm.sudo("echo -e #{content.inspect} > /etc/synthetic.conf")
-                  if @apply_firmlinks[:bootstrap]
+                  if @apply_firmlinks[machine.id][:bootstrap]
+                    # require "pry-byebug"; binding.pry
+                    if machine.guest.capability("darwin_major_version").to_i < 20
```

Comment:
Set the `20` value as a constant (like `MACOS_BIGSUR_DARWIN_VERSION` or w/e) and use it in place of the static value. It's just helpful as it provides better context on what the value means.